lib/mocha/configuration.rb

```diff
@@ -43,7 +45,8 @@ class Configuration
       :stubbing_non_public_method => :allow,
       :stubbing_method_on_nil => :prevent,
       :display_matching_invocations_on_failure => false,
-      :reinstate_undocumented_behaviour_from_v1_9 => true
+      :reinstate_undocumented_behaviour_from_v1_9 => true,
```

Comment:
Hmm this seems like something we should deprecate as well? Maybe a separate PR

Reply:
Yes, I recently opened #560 to start capturing things I'd like to deprecate before releasing v2.0.